src/pyodide/internal/python.js

```diff
@@ -186,9 +186,13 @@ function loadDynlib(Module, path, wasmModuleData) {
  * there.
  */
 function preloadDynamicLibs(Module) {
+  let SO_FILES_TO_LOAD = SITE_PACKAGES_SO_FILES;
+  if (DSO_METADATA?.settings?.baselineSnapshot) {
+    SO_FILES_TO_LOAD = [[ '_lzma.so' ]];
```

Comment:
is this definitely safe? Should we maybe not preload anything just to be extra cautious?

Reply:
This gets imported by something in `SNAPSHOT_IMPORTS` I'm not really sure why. But because we use it in the snapshot, we'll crash without it.